draft-foudil-securitytxt.md

```diff
@@ -70,8 +76,7 @@ can have an unlimited number of fields. It is important to note that
 you need a separate line for every field. One MUST NOT chain multiple
 values for a single directive. Everything MUST be in a separate field.
 
-A security.txt file only applies to the domain, subdomain, IPv4 or IPv6 address
-it is located in.
+A security.txt file only applies to the domain in the URI used to retrieve it, not to any of its subdomains or parent domains.
```

Comment:
What about IP addresses? Only for that IP and not anything in the same subnet, above or below? Or is that self evident?

What about file systems?